android/src/main/java/io/invertase/firebase/messaging/RNFirebaseMessaging.java

```diff
@@ -59,6 +59,11 @@ public void getToken(Promise promise) {
     promise.resolve(token);
   }
 
+  @ReactMethod
+  public void getAPNSToken(Promise promise) {
```

Comment:
We can remove this and add an `if (isAndroid) return null;` line in the JS method, native code changes not required then 👍